modules/rabbitmq/src/test/java/org/testcontainers/containers/RabbitMQContainerTest.java

```diff
@@ -0,0 +1,224 @@
+ */
+public class RabbitMQContainerTest {
+
+    public static final String DEFAULT_TAG = "3.7-management-alpine";
+    public static final int DEFAULT_AMQPS_PORT = 5671;
+    public static final int DEFAULT_AMQP_PORT = 5672;
+    public static final int DEFAULT_HTTPS_PORT = 15671;
+    public static final int DEFAULT_HTTP_PORT = 15672;
+
+    @Test
+    public void shouldCreateRabbitMQContainer() {
+        try (RabbitMQContainer container = new RabbitMQContainer()) {
+
+            assertThat(container.getDockerImageName()).isEqualTo("rabbitmq:" + DEFAULT_TAG);
+            assertThat(container.getAdminPassword()).isEqualTo("guest");
+            assertThat(container.getAdminUsername()).isEqualTo("guest");
+
+            container.start();
+            container.waitUntilContainerStarted();
+
+            assertThat(container.getAmqpsUrl()).isEqualTo(
+                    String.format("amqps://%s:%d", container.getContainerIpAddress(), container.getMappedPort(DEFAULT_AMQPS_PORT)));
+            assertThat(container.getAmqpUrl()).isEqualTo(
+                    String.format("amqp://%s:%d", container.getContainerIpAddress(), container.getMappedPort(DEFAULT_AMQP_PORT)));
+            assertThat(container.getHttpsUrl()).isEqualTo(
+                    String.format("https://%s:%d", container.getContainerIpAddress(), container.getMappedPort(DEFAULT_HTTPS_PORT)));
+            assertThat(container.getHttpUrl()).isEqualTo(
+                    String.format("http://%s:%d", container.getContainerIpAddress(), container.getMappedPort(DEFAULT_HTTP_PORT)));
+
+            assertThat(container.getHttpsPort()).isEqualTo(container.getMappedPort(DEFAULT_HTTPS_PORT));
+            assertThat(container.getHttpPort()).isEqualTo(container.getMappedPort(DEFAULT_HTTP_PORT));
+            assertThat(container.getAmqpsPort()).isEqualTo(container.getMappedPort(DEFAULT_AMQPS_PORT));
+            assertThat(container.getAmqpPort()).isEqualTo(container.getMappedPort(DEFAULT_AMQP_PORT));
+
+            assertThat(container.getLivenessCheckPortNumbers()).containsExactlyInAnyOrder(
+                    container.getMappedPort(DEFAULT_AMQP_PORT),
+                    container.getMappedPort(DEFAULT_AMQPS_PORT),
+                    container.getMappedPort(DEFAULT_HTTP_PORT),
+                    container.getMappedPort(DEFAULT_HTTPS_PORT)
+            );
+        }
+    }
+
+    @Test
+    public void shouldCreateRabbitMQContainerWithTag() {
+        try (RabbitMQContainer container = new RabbitMQContainer(DEFAULT_TAG)) {
+            assertThat(container.getDockerImageName()).isEqualTo("rabbitmq:" + DEFAULT_TAG);
+        }
+    }
+
+    @Test
+    public void shouldCreateRabbitMQContainerWithExchange() throws IOException, InterruptedException {
+        try (RabbitMQContainer container = new RabbitMQContainer()) {
+            container.withExchange("test-exchange", "direct");
+
+            container.start();
+            container.waitUntilContainerStarted();
```

Comment:
`waitUntilContainerStarted` is executed by `.start()`, no need to do it here

Reply:
(same applies to other usages)

Reply:
Good catch. Done